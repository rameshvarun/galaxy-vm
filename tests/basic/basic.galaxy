// Basic galaxy test.

void Main(int argc, string[100] argv) {
  ConsoleLog("Basic Galaxy Test...");
  TestFunctionA();
}

void TestFunctionA() {
  ConsoleLog("Test Function A...");
}
