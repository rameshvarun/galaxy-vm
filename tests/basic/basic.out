Basic Galaxy Test...
Test Function A...
